Makefile

```diff
@@ -58,13 +61,20 @@ all: install
 	go-bindata ${GO_BINDATA_FLAGS} -nometadata -o statics/bindata.go -pkg=statics -ignore=bindata.go statics/* statics/css/images/* statics/js/vendor/* statics/js/components/* ${EXTRABINDATA}
 	gofmt -w -s statics/bindata.go
 
+define govendor_do
+$(GOVENDOR) $1 \
+	-ldflags="-X $(SKYDIVE_GITHUB)/version.Version=${VERSION}" \
+	${GOFLAGS} -tags="${BUILDTAGS} ${GOTAGS}" ${VERBOSE_FLAGS} \
+	+local
+endef
+
 .compile:
-	${GOPATH}/bin/govendor install -ldflags="-X github.com/skydive-project/skydive/version.Version=${VERSION}" ${GOFLAGS} -tags="${BUILDTAGS} ${GOTAGS}" ${VERBOSE_FLAGS} +local
+	$(call govendor_do,install)
 
 install: govendor genlocalfiles dpdk.build contribs .compile
 
 build: govendor genlocalfiles dpdk.build contribs
-	${GOPATH}/bin/govendor build -ldflags="-X github.com/skydive-project/skydive/version.Version=${VERSION}" ${GOFLAGS} -tags="${BUILDTAGS} ${GOTAGS}" ${VERBOSE_FLAGS} +local
+	$(call govendor_do,build)
 
 static: govendor genlocalfiles
```

Comment:
could you please update the static target as well
with $(call govendor_do, install, extraLDFlags)

Reply:
@nplanel - because of the differences it was not feasible to extend `govendor_do` - but I did small code improvement (to remove duplication and make it easier to maintain going forward)